build-logic/build_properties.gradle.kts

```diff
@@ -13,8 +13,7 @@ extra.apply {
      * CI. Read more about our versioning here:
      * https://simprints.atlassian.net/wiki/spaces/KB/pages/1761378305/Releasing+Simprints+ID
      *
-     * Dev version >= 2023.4.1 is required for receiving encryption Tokens from BFSID [CORE-2502]
-     * Dev version >= 2023.4.0 is required for receiving new fingerprint configurations  [CORE-3033]
+     * Dev version >= 2024.2.1 is required for receiving biometric sdk age restrictions
      */
     set("VERSION_NAME", "2024.1.1")
```

Comment:
You wrote that the dev version should be >= 2024.2.1, but you set 2024.1.1 you will not get the age configuration